lib/commands/ls.js

```diff
@@ -96,9 +96,11 @@ class LS extends ArboristWorkspaceCmd {
       }
 
       if (edge.from.isProjectRoot) {
-        return edge.to &&
-          edge.to.isWorkspace &&
-          wsNodes.includes(edge.to.target)
+        return (edge.to
```

Comment:
This is getting a little unweildy, maybe a comment is in order, or two if statements?

Reply:
done

Reply:
Yes